testplan/runnable/interactive/base.py

```diff
@@ -839,7 +840,11 @@ def _setup_http_handler(self):
             ihandler=self, port=self.cfg.http_port
         )
         http_handler.cfg.parent = self.cfg
-        http_handler.setup()
+
+        # Mute flask_restx warning.
+        with warnings.catch_warnings():
```

Comment:
How about to put a time limit for suppression. If we would not upgrade restx in let say 6 month this should come back even with an extra annoying warning :)

Reply:
Let me reply internally.